app/src/main/java/com/sun/basic_japanese/flashcard/eachpage/FlashCardPageFragment.kt

```diff
@@ -62,6 +84,19 @@ class FlashCardPageFragment : Fragment(), FlashCardPageContract.View {
         }
     }
 
+    private fun setEventClick() {
+        buttonPlayAudio.setOnClickListener {
+            TODO()
+        }
+        switchRemember.setOnClickListener {
+            flashCard.alphabet?.let {
+                if (switchRemember.isChecked) it.remember = REMEMBERED
```

Comment:
đoạn này cũng tương tự 
```suggestion
               it.remember =  if (switchRemember.isChecked) REMEMBERED else 
```